keyboards/db/db63/README.md

```diff
@@ -0,0 +1,21 @@
+# DB63
+
+A 65% keyboard hotswap with RGB that runs ps2avrgb natively, with USB C, RGB underglow and backlight.
+Firmware is still in **beta**
+
+* Keyboard Maintainer: QMK Community
+* Hardware Supported: 
+* Hardware Availability: 
+* [Keyboard layout](http://www.keyboard-layout-editor.com/#/gists/dadea703fc8bfc87dc7c480de9f3ef38)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make db/db63:default
+
+Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
+
+    make db/db63:default:flash
+
+**Reset Key**: Hold down the key located at *K00*, commonly programmed as *left control* while plugging in the keyboard. Another trick to enable the board in reset mode is using BootMapper Client->Options->Set Bootloader (this is only available for the first times before flashing QMK firmware)
```

Comment:
Is the stock Reset key locked to B5 × A0? Because that would be `K40` and not `K00`. Otherwise, the `*left control*` here should be `*Escape*`. which *is* `K00`.

This paragraph should almost definitely be expanded if the flashing procedure changes after QMK is installed on the board.

Reply:
It's the *Escape* key, i will update this paragraph